build.sh

```diff
@@ -3,8 +3,7 @@
 ./compile-translations.sh
 
 python3 asknot-ng.py \
-	./templates/index.html \
-	./questions/fedora.yml \
-	./l10n/fedora/locale \
-	-s ./static \
-	-t fedora
+	templates/index.html \
+	questions/arch.yml \
+	l10n/arch/locale \
+	--theme arch
```

Comment:
archlinux*

Reply:
yeah i changed it from arch.yml to archlinux.yml thats it right?
or Should i change the directory names too?